packages/nasm.vm/nasm.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>nasm.vm</id>
-    <version>2.16.1.20240217</version>
+    <version>2.16.1.20240331</version>
     <authors>NASM Authors</authors>
     <description>Netwide Assembler</description>
     <dependencies>
       <dependency id="common.vm" />
-      <dependency id="nasm" version="[2.16.1.20221231]" />
+      <dependency id="nasm" version="[2.16.01.20221231]" />
```

Comment:
I think this is correct as it is, as it is the version chocolatey returns:
```
PS C:\Windows\system32 > choco find -er nasm
Chocolatey v2.2.2
nasm|2.16.1.20221231
FLARE-VM 04/03/2024 09:14:03
```
This is also what the update script uses. Is this causing any error? Can you please share more details?

Reply:
It seems to be a problem in the CLI, but not when installing the package:
![image](https://github.com/mandiant/VM-Packages/assets/16052290/cc588ef6-f05a-4ba4-8921-1e1f2a236bad)

The bot updated to 2.16.01.20221231 in https://github.com/mandiant/VM-Packages/commit/a3761cd153f64f990235bccd66cf78d245e7f6a3 (Jun 1, 2023) and to 2.16.1.20221231 in https://github.com/mandiant/VM-Packages/commit/cd949932263a7c2ce455d5d1bc6fdf71bae788c2 (Jun 14, 2023). It looks like this was introduced in Chocolatey in https://github.com/chocolatey/choco/pull/3175 and it is the intended new behaviour (normalized version). I think our code should use what is consistent with choco output (as otherwise the bot will change it back) and that the issue is that the CLI URLs do not supported normalized versions. I think you should report to chocolatey the fact that the URLs do not work with normalized versions, being inconsistent with command line output.

Reply:
Thank for the answer, I will look to fix this on my side and I will remove this from the PR